app/src/main/res/layout/problem_detail.xml

```diff
@@ -1,141 +1,150 @@
-                android:textColor="#DE000000"
-                android:textSize="14sp" />
+            <include
+                android:id="@+id/problem_image_pager_layout"
+                android:elevation="3dp"
+                layout="@layout/problem_photo_gallery" />
 
-            <TextView
-                android:id="@+id/problem_entry_date"
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:layout_alignParentRight="true"
-                android:fontFamily="sans-serif-regular"
-                android:textColor="#6d6d6d"
-                android:textSize="14sp" />
-
-        </RelativeLayout>
-
-        <TextView
-            android:id="@+id/problem_description"
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:lineSpacingExtra="4sp"
-            android:paddingLeft="24dp"
-            android:paddingRight="24dp"
-            android:paddingTop="8dp"
-            android:paddingBottom="24dp"
-            android:textColor="#DE000000"
-            android:textSize="14sp" />
-
-        <RelativeLayout
-            android:id="@+id/problem_answer_block"
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:background="#f4f4f4"
-            android:padding="24dp">
-
-            <TextView
-                android:id="@+id/problem_answer_label"
-                android:layout_width="wrap_content"
+            <RelativeLayout
+                android:layout_width="match_parent"
                 android:layout_height="wrap_content"
-                android:fontFamily="sans-serif-medium"
-                android:text="@string/answer"
-                android:textColor="#DE000000"
-                android:textSize="14sp" />
-
-            <TextView
-                android:id="@+id/problem_answer_date"
-                android:layout_width="wrap_content"
+                android:paddingBottom="24dp"
+                android:paddingLeft="24dp"
+                android:paddingRight="24dp"
+                android:paddingTop="18dp">
+
+                <TextView
+                    android:id="@+id/problem_title"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_toLeftOf="@+id/problem_status"
+                    android:paddingRight="16dp"
+                    android:fontFamily="sans-serif-regular"
+                    android:textColor="#DE000000"
+                    android:textSize="24sp" />
+
+                <TextView
+                    android:id="@+id/problem_status"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_alignParentRight="true"
+                    android:layout_marginTop="7dp"
```

Comment:
Avoid margin `dp` values which can't be divided by 8. https://material.google.com/layout/metrics-keylines.html#